sdk/schemaregistry/schema-registry/CHANGELOG.md

```diff
@@ -1,14 +1,11 @@
 # Release History
 
-## 1.2.1 (Unreleased)
+## 1.3.0-beta.1 (Unreleased)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Adds support for JSON and custom schema formats. A schema of the custom format can be in a format that is not currently supported.
+- Adds `KnownSchemaFormats` enum that has `Avro`, `json`, and `custom` values.
```

Comment:
just checking that the case on these enums is intentionally different?
I think .NET has `JSON`. In Python, we're capitalizing all of these (`Avro`, `Json, and `Custom`).

Reply:
I think it is Json in .NET.

Reply:
addressed in https://github.com/Azure/azure-sdk-for-js/pull/23575/commits/0705a8727bb9d8e03a7727259662b2b58c617905

Reply:
and in https://github.com/Azure/azure-sdk-for-js/pull/23575/commits/3ef59308fddc806dc05792bbdb062b15e9c0fbbe